libsql/src/hrana/mod.rs

```diff
@@ -33,8 +38,62 @@ struct Cookie {
 }
 
 pub trait HttpSend<'a>: Clone {
-    type Result: Future<Output = Result<ServerMsg>> + 'a;
-    fn http_send(&'a self, url: String, auth: String, body: String) -> Self::Result;
+    type Result: Future<Output = Result<HttpBody>> + 'a;
+    fn http_send(&'a self, url: &'a str, auth: &'a str, body: String) -> Self::Result;
+}
+
+#[cfg(feature = "wasm")]
+pub type ByteStream = Box<dyn Stream<Item = Result<Bytes>> + Unpin>;
+
+#[cfg(not(feature = "wasm"))]
+pub type ByteStream = Box<dyn Stream<Item = Result<Bytes>> + Send + Unpin>;
```

Comment:
We need some way to work around this kind of issues - `Send`/`Sync` is only allowed in certain contexts but we cannot really make it config flag-only dependent for the sake of `cargo check --all-features`.